test/system/index.spec.ts

```diff
@@ -239,7 +269,7 @@ describe("system tests", () => {
   test("inspect image with sha@256 " + "ubuntu@sha256", async () => {
     const imgName = "ubuntu";
     const imgSha =
-      "@sha256:945039273a7b927869a07b375dc3148de16865de44dec8398672977e050a072e";
+      "@sha256:eb5d7eda6804359e4fc5223a31a2d9caa4c8ea590b14060d81c8bc05b22ca04e";
```

Comment:
If you're wondering about this change - it's the exact same image - I learned some interesting things about image digests: https://github.com/moby/moby/issues/43188#issuecomment-1192448412